packages/firebase_auth/README.md

```diff
@@ -20,7 +20,7 @@ To use the firebase_auth plugin, follow the [plugin installation instructions](h
 Enable the Google services by configuring the Gradle scripts as such.
 
 1. Add the classpath to the `[project]/android/build.gradle` file.
-```gradle
+```groovy
```

Comment:
Do you know what's the difference in how Github treats groovy vs gradle?
From the preview it looks like this block is a little  prettier with gradle, e.g `dependencies` is colored:

For reference:

That's how it looks with gradle:
```gradle
dependencies {
  // Example existing classpath
  classpath 'com.android.tools.build:gradle:3.2.1'
   // Add the google services classpath
   classpath 'com.google.gms:google-services:3.2.1'
 }
```

That's how it looks with groovy:
```groovy
dependencies {
  // Example existing classpath
  classpath 'com.android.tools.build:gradle:3.2.1'
   // Add the google services classpath
   classpath 'com.google.gms:google-services:3.2.1'
 }
```

Reply:
You're correct! Sorry about that.